tests/suites/test_suite_psa_crypto.function

```diff
@@ -8084,3 +8084,327 @@ exit:
     PSA_DONE();
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_ECJPAKE_C */
```

Comment:
Why `ECJPAKE_C` rather than `PSA_WANT_ALG_ECJPAKE`? (Same for the other test functions.)

Reply:
Yep, will clean the dependencies